src/ui/settings/language.js

```diff
@@ -11,8 +11,15 @@ const LanguageSettings = () => {
   const { i18n } = useTranslation();
 
   return (
-    <SettingsSection title="Language">
+    <SettingsSection title={i18n.t('settings-general-header')}>
+      <label
+        htmlFor='studio-lang'
+        sx={{ variant: 'styles.label' }}
+      >
+        {i18n.t('settings-language-label')}
+      </label>
       <select
+        id="studio-lang"
```

Comment:
We actually explicitly kept "Language" untranslated. If you set Studio to some weird language, you still want to find the language selection, and we figured most people know what "language" means. We plan on improving this a lot in the future via some redesign.

So, could you remove this part of the change (including the new translation strings). 

Regarding adding a label to this and other drop-down menus: is this mainly for accessibility? I'm not quite sure what this change is for. I'm fully on board with the change for the audio and video device (as it doesn't change anything about what's visible), but I'm not sure it's useful in this language case?

Reply:
Ok ! I make sense, yes ;) Maybe we can keep both : translated string + "Language" between parenthesis ?

Adding the labels is mandatory for accessibility yes.

Reply:
to evaluate accessibility level of a web page, I recommend the "Wave" tool : https://wave.webaim.org/ which can be installed as a plugin in your web browser.

Reply:
I do know Wave and have it permanently installed actually. I guess I just never checked these Studio things, probably because due to the lack of time, but likely also because not long ago, Studio had way more serious accessibility concerns :D